src/main/java/racingcar/Application.java

```diff
@@ -1,26 +1,21 @@
 package racingcar;
 
 
-import racingcar.common.UserInput;
+import racingcar.controller.RacingController;
 import racingcar.domain.Cars;
-import racingcar.domain.Judgement;
-import racingcar.domain.Racing;
-import racingcar.domain.movable.RandomForwardStrategy;
+import racingcar.service.RacingService;
+import racingcar.view.InputView;
+import racingcar.view.ResultView;
 
 public class Application {
 
-    private static final int START_FROM_ZERO = 0;
-
     public static void main(String[] args) {
-        Racing racing = Racing.instance();
-        UserInput userInput = UserInput.instance();
-        Judgement judgement = Judgement.instance();
-        Cars cars = Cars.of(userInput.getSplitUserInput(), new RandomForwardStrategy());
-        for (int time = START_FROM_ZERO; time < userInput.getRacingTime(); time++) {
-            racing.drive(cars.getCars());
-            judgement.carsStatement(cars.getCars());
-        }
-        System.out.println(judgement.judge(cars.getCars()));
+        RacingService racingService = RacingService.from(Cars.instance());
+        RacingController racingController = new RacingController(racingService,
+            InputView.instance(), ResultView.instance());
+        racingController.setUpRacing();
+        racingController.startRacing();
+        System.out.println(racingController.finishRacing());
```

Comment:
```suggestion
        racingController.finishRacing();
```
이렇게 되면 좀더 깔끔하게 표현할 수 있을 것 같습니다.